Doc/library/html.parser.rst

```diff
@@ -175,16 +175,16 @@ implementations do nothing (except for :meth:`~HTMLParser.handle_startendtag`):
    This method is called to process decimal and hexadecimal numeric character
    references of the form ``&#NNN;`` and ``&#xNNN;``.  For example, the decimal
    equivalent for ``&gt;`` is ``&#62;``, whereas the hexadecimal is ``&#x3E;``;
-   in this case the method will receive ``'62'`` or ``'x3E'``.  This method
+   in this case the *name* parameter will be ``'62'`` or ``'x3E'``.  This method
```

Comment:
Why is it necessary to mention the parameter name explicitly? The old wording is a little more concise and doesn't seem ambiguous to me. This applies to most of your other changes too.

Reply:
Because the [page](https://docs.python.org/3/library/html.parser.html) already mentions parameter names for some methods:

- `HTMLParser.handle_decl`:

  > The *decl* parameter will be the entire contents of the declaration inside the `<!...>` markup (e.g. `'DOCTYPE html'`).

- `HTMLParser.handle_pi(data)`:

  > The *data* parameter will contain the entire processing instruction.

- `HTMLParser.unknown_decl(data)`:

  > The *data* parameter will be the entire contents of the declaration inside the `<![...]>` markup.

Reply:
OK. I'll leave this up to the maintainer of the module, but personally I don't think consistency is too important here.

Reply:
Seems unneeded to me too.